packages/rocketchat-livechat/.app/i18n/nl.i18n.json

```diff
@@ -25,21 +25,21 @@
   "Request_video_chat": "Vraag videochat aan",
   "Select_a_department": "Selecteer een afdeling",
   "Switch_department": "Van afdeling wisselen",
-  "Department_switched": "Afdeling geschakeld",
+  "Department_switched": "Afdeling gewisseld",
   "Send": "sturen",
   "Skip": "Overslaan",
   "Start_Chat": "Start Chat",
   "Survey": "Enquête",
-  "Survey_instructions": "Beoordeel elke vraag naar mate uw tevredenheid, 1 betekent dat u helemaal ontevreden  bent en 5 betekent volledig tevreden.",
+  "Survey_instructions": "Beoordeel elke vraag naar mate uw tevredenheid, 1 betekent dat u helemaal ontevreden bent en 5 betekent volledig tevreden.",
   "Thank_you_for_your_feedback": "Hartelijk dank voor uw feedback",
-  "Thanks_We_ll_get_back_to_you_soon": "Bedankt! We zullen contact met u binnenkort.",
-  "transcript_sent": "Transcript verzonden",
+  "Thanks_We_ll_get_back_to_you_soon": "Bedankt! We zullen contact met u opnemen binnenkort.",
```

Comment:
-> We zullen binnenkort contact met u opnemen?

Reply:
@wampiedriessen Sorry, what is the issue here?

Reply:
"Bedankt! We zullen binnenkort contact met u opnemen"
Would be a better phrasing

Reply:
```suggestion
  "Thanks_We_ll_get_back_to_you_soon": "Bedankt! We zullen binnenkort contact met u opnemen.",
```